project-koku/koku
# COST-645: Download GCP report using Google BigQuery

How to test

Run `make gcp-source gcp_name=test_gcp_source` and then check to see if the csv file was created.
